### DISCUSSION TITLE
Current hardware being used

### DISCUSSION THREAD

**Discussion by reedy-au** [awawa-dev/HyperHDR#565] [Q&A]
I was about to buy some more esp32 boards and I was going to ask awawa what hardware is currently being used for development. 

As I've had bad experiences with some boards and some boards with chips not catering for the recommended uart speeds for best performance, I thought starting a new thread that could be on going and kept up to date for users may help.

Awawa can you advise what boards you are current developing on please.

This is the esp I'm currently using for all deployments: https://www.aliexpress.com/item/32836372640.html

**Comment by awawa-dev** [comment]
Hi
I currently have 4 different Rp2040's I've been using for development, all work fine. It's all in one SoC, so it's hard to spoil it with some "improvements" of the manufacturer.. in simplicity similar to esp32-s2 mini but more powerful, user friendly, SDK quality at least several times better than ESP espressif (yes they lack wifi but esp32-s2 also doesn't have a stable wifi led driver, wled support has been in the works for a few years). And I have esp32 MH-ET in my home system, I consider it as most mature and stable solution for now. Although in practice the Rp2040 generates a more accurate signal, it is rather an advantage for purists like me: esp boards also work without problems.

**Comment by reedy-au** [comment]
Awesome. At this point we know the following hardware definitely works with hyperhdr.

Controller:
- https://www.aliexpress.com/item/1005004967926448.html
- https://www.aliexpress.com/item/32836372640.html

Splitter:
- https://www.amazon.com/Splitter-l-b-y-Vision-Atmos-Scaler/dp/B07VP37KMB/
- https://www.amazon.de/HDMI-Matrix-Out-4K-60Hz/dp/B087JCN28V/ref=sr_1_7_sspa?__mk_de_DE=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=41UVWYA52RLN
- https://www.aliexpress.com/item/1005004685856434.html
- https://www.tesmart.com/products/tesmart-hdmi-splitter-1-in-2-out-support-4k-60hz-hdr-10-hdcp-2-2?variant=43617192116436



Grabber/Capture:
- https://www.aliexpress.com/item/1005004990708725.html
- https://www.amazon.com/gp/product/B097DKNS1M/
- Old:
  - https://www.aliexpress.com/item/4001044845208.html

Should we make a list? :smiley:

**Comment by reedy-au** [comment]
Hi @awawa-dev 

How are you, been away for a a while. I'm looking at updating hyperhdr to v19 from v18 are you still using these boards: https://www.aliexpress.com/item/32836372640.html

Hope you are well.

Thanks

**Reply by awawa-dev** [reply]
Hi
Yes, I still use this esp32 model together with HyperSPI in my home setup, but if you are looking for serial port solution then currently a better choice will be Raspberry Pi Pico. Especially since some models have a built-in level shifter and you don't have to bother with using any flashers: https://github.com/awawa-dev/HyperSerialPico It has been tested by users for several months and no major problems have been found so far. HyperSPI version for Pico is underway but it's hard to say when it will be ready.
